lib/internal/source_map/source_map.js

```diff
@@ -175,24 +191,18 @@ class SourceMap {
     const entry = this.#mappings[first];
     if (!first && entry && (lineNumber < entry[0] ||
         (lineNumber === entry[0] && columnNumber < entry[1]))) {
-      return null;
-    }
-    return entry;
-  }
-
-  /**
-   * @param {string} sourceURL of the originating resource
-   * @param {number} lineNumber in the originating resource
-   * @return {Array}
-   */
-  findEntryReversed(sourceURL, lineNumber) {
```

Comment:
`findEntryReversed` was not being used anywhere in our codebase, and seemed like it was broken logic. I'd rather add it back in to the class in the future if it's needed by folks, and keep the API minimal.